declarations/LoaderContext.d.ts

```diff
@@ -0,0 +1,185 @@
+import type { RawSourceMap } from "source-map";
+import type { Schema } from "schema-utils/declarations/ValidationError";
```

Comment:
Maybe we can export `ValidationError` to avoid using package directly, I am afraid it can be changed in future

Reply:
I think we can get the `Schema` from the root types too: `Parameters<import("schema-utils").validate>[1]`